TrafficSimulator/Assets/Utilities/DayNightManager.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Simulation
+{
+    public enum SunMode
+    {
+        Time,
+        Day,
+        Night
+    }
+
+    public class DayNightManager : MonoBehaviour
+    {
+        public SunMode _sunMode = SunMode.Time;
```

Comment:
```suggestion
        public SunMode _sunMode = SunMode.Day;
```

Reply:
Default should be Day so the scenes will look the same regardless of what the time is when you open them